src/pages/settings/profile/index.jsx

```diff
@@ -82,19 +92,21 @@ const ProfileSettings = () => {
   if (user) {
     return (
       <>
-        <PageHeader
+        <Header
           title='Profile'
           extra={(
             <FeedbackButton />
           )}
         />
-        <Card>
-
-          <Row type='flex' justify='center' align='center'>
-            <Col style={{ width: '40%' }}>
+        <Space direction='vertical' style={{ width: '100%', padding: '20px', background: ' white' }}>
+          <Row type='flex'>
+            <Col xl={{ span: 12, offset: 6 }} span={24}>
 
-              <Form layout='horizontal'>
-                <h2>Profile settings:</h2>
+              <Form
+                layout='horizontal'
+                {...formItemLayout}
```

Comment:
The linter is showing an error, can you add the required ignore for this line?

Reply:
I just added the properties without the variable